content/doc/applications/docker/_index.md

```diff
@@ -204,3 +204,18 @@ We provide a few examples of dockerized applications on Clever Cloud.
 {{% content/more-config %}}
 
 {{% content/url_healthcheck %}}
+
+### Commonly encountered issues
+
+{{% details title="I can't reach my app port" closed="true" %}}
+
+You probably want to use a specific port for their apps. However, you can't directly reach for that port, as your request will go through Clever Cloud's reverse proxies, then through the VM.
+
+Here are the steps:
+* Create a TCP redirection that will allow entry through our load balancers, that will come out as 4040 or 8080 on the Docker. You can find a guide here:
```

Comment:
`coming out on the Docker` isn't clear enough. Which workflow are you trying to describe here?

```suggestion
* Create a TCP redirection to allow entry through Clever Cloud load balancers, with port 4040 or 8080 coming out on the Docker. Check [this guide](/doc/administrate/tcp-redirections) for more info.
```